docs/en/operations/settings/settings.md

```diff
@@ -3155,7 +3155,7 @@ Possible values:
 - Positive integer.
 - 0 or 1 — Disabled. `SELECT` queries are executed in a single thread.
 
-Default value: `16`.
+Default value: the number of physical CPU cores.
```

Comment:
to avoid listing all the details we could simply say it defaults to `max_threads`

Reply:
done